docs/source/troubleshooting/basic_chatops_troubleshooting.rst

```diff
@@ -30,11 +30,11 @@ These are the issues that users usually face with ChatOps:
 Your bot is not Online:
 -----------------------
 
-You've installed StackStorm, but Hubot didn't come online in your ChatOps client (Slack, HipChat, etc.)
+You've installed StackStorm, but Hubot did not come online in your ChatOps client (Slack, HipChat, etc.)
 
 **Possible reasons:**
 
-- Incorrect adapter settings. Check ``/opt/stackstorm/chatops/st2chatops.env`` for settings.
+- Incorrect adapter settings. Check the settings using ``/opt/stackstorm/chatops/st2chatops.env`` and verify if correct.
```

Comment:
`st2chatops.env` is a configuration file, not a command. So I think it might be more accurate to say "Check the settings in /opt/stackstorm/chatops/st2chatops.env, and verify if they are correct" ?

What do you think?